roles/oradb-manage-db/tasks/tnsnames.yml

```
@@ -22,7 +22,7 @@
     create: True
     group: "{{ oracle_group }}"
     owner: "{{ oracle_user }}"
-    state: present
+    state: "{{ tnsinst.state }}"
```

We should add a default here:
`    state: "{{ tnsinst.state | default('present')}}"`

Otherwise we brake existing configurations with missing state in inventory.

---

```suggestion
    state: "{{ tnsinst.state | default('present') }}"
```
That is indeed a very important thing. Always good to have someone else look over the code.